is.js

```diff
@@ -176,6 +176,15 @@
         return toString.call(value) === '[object RegExp]';
     };
 
+    // is a given value a Map?
+    is.map = function(value) {
+        // checking stringified constructor in case toString() yields '[object Object]'
+        // instead of '[object Map]'
+        var mapRegex = /^function\s{1,}Map\(\)/;
+        var constructorStr = (value.constructor) ? value.constructor.toString() : '';
```

Comment:
Let's not assume the constructor as `toString()` (it likely does but still). Instead `(value.constructor + '')` would do.